tests/lib/rules/no-node-access.test.ts

```diff
@@ -173,14 +173,62 @@ ruleTester.run(RULE_NAME, rule, {
 				user.click(buttonText);
       `,
 			},
-			...EVENTS_SIMULATORS.map((simulator) => ({
+			{
+				code: `
+				import userEvent from '@testing-library/user-event';
+        import { screen } from '${testingFramework}';
+
+        const buttonText = screen.getByText('submit');
+				const userAlias = userEvent.setup();
+				userAlias.click(buttonText);
+      `,
+			},
+			{
+				code: `
+				import userEvent from '@testing-library/user-event';
+        import { screen } from '${testingFramework}';
+
+        const buttonText = screen.getByText('submit');
+				userEvent.setup().click(buttonText);
+      `,
+			},
+			{
 				code: `
+				import userEvt from '@testing-library/user-event';
         import { screen } from '${testingFramework}';
 
         const buttonText = screen.getByText('submit');
-				${simulator}.click(buttonText);
+				const userAlias = userEvt.setup();
+				userAlias.click(buttonText);
+      `,
+			},
+			{
+				code: `
+				import userEvt from '@testing-library/user-event';
+        import { screen } from '${testingFramework}';
+
+        const buttonText = screen.getByText('submit');
+				userEvt.click(buttonText);
+      `,
+			},
+			{
+				code: `
+        import { screen, fireEvent as fe } from '${testingFramework}';
+
+        const buttonText = screen.getByText('submit');
+				fe.click(buttonText);
       `,
-			})),
+			},
+			{
+				settings: { 'testing-library/utils-module': 'test-utils' },
+				code: `
+				// case: custom module set but not imported using ${testingFramework} (aggressive reporting limited)
+        import { screen, fireEvent as fe } from 'test-utils';
+
+        const buttonText = screen.getByText('submit');
+				fe.click(buttonText);
+      `,
+			},
```

Comment:
One last thing, sorry. Can you add a valid test to check that the issue reported in #1032 has been fixed?